src/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -200,29 +145,26 @@ export const Interactive: StoryObj<Args> = {
   ],
 };
 
-export const InteractiveDisabledButton: StoryObj<Args> = {
+export const InteractiveDisabled: StoryObj<Args> = {
   args: {
     duration: undefined,
   },
   render: (args) => (
-    <div
-      className={clsx(
-        styles['trigger--spacing-top'],
-        styles['trigger--spacing-left'],
-      )}
+    <Tooltip
+      align="top"
+      childNotInteractive
+      duration={args.duration}
+      text={defaultArgs.text}
     >
-      <Tooltip
-        align="top"
-        childNotInteractive
-        duration={args.duration}
-        text={defaultArgs.text}
-      >
-        <Button disabled variant="primary">
-          Tooltip trigger
-        </Button>
-      </Tooltip>
-    </div>
+      <div className="fpo  p-1">&bull;</div>
+    </Tooltip>
   ),
+  parameters: {
+    chromatic: {
+      diffThreshold,
+      diffIncludeAntiAliasing: false,
+    },
+  },
```

Comment:
Should also be able to remove these, since they're already defined at the component level (this file's default export)